.github/workflows/test.yml

```diff
@@ -432,6 +428,9 @@ jobs:
         with:
           node-version: ${{ vars.NODE_VERSION }}
 
+      - name: Clear npm cache
```

Comment:
This should no longer be required unless we want to reclaim disk space, latest `NPM` is self healing
Reference: https://docs.npmjs.com/cli/v10/commands/npm-cache